server/models/User.js

```diff
@@ -24,5 +24,42 @@ const UserSchema = new Schema({
 	}
 });
 
+// Add a new user
+UserSchema.statics.addNewUser = function(firstName, lastName, email, password){
+	return this.insert({
```

Comment:
I'm pretty sure this method doesn't exist in mongoose, there's .create() for one document and .insertMany() for many documents. I would recommend create for this use